<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPartida</class>
 <widget class="QMainWindow" name="TelaPartida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Truco Paulista</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #2F434F;</string>
   </property>
   <widget class="QPushButton" name="btnCarta1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>120</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCarta2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>120</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCarta3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>120</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTruco">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pedir Truco</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblManilha">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>90</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>Manilha:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblPontos">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pontos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTruco">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>112</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rodada valendo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMesa">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>400</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string>Cartas na mesa:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="pontoNos1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/circulo_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pontoNos2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/circulo_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pontoNos3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/circulo_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mesa">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>mesa.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pontoEles1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/circulo_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pontoEles3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/circulo_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pontoEles2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>250</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/circulo_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador1_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador1_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador1_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador2_1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador2_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador2_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador3_1">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador3_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartaJogador3_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sairSala">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sair da sala</string>
    </property>
   </widget>
   <zorder>mesa</zorder>
   <zorder>btnCarta1</zorder>
   <zorder>btnCarta2</zorder>
   <zorder>btnCarta3</zorder>
   <zorder>btnTruco</zorder>
   <zorder>lblManilha</zorder>
   <zorder>lblPontos</zorder>
   <zorder>lblTruco</zorder>
   <zorder>lblMesa</zorder>
   <zorder>pontoNos1</zorder>
   <zorder>pontoNos2</zorder>
   <zorder>pontoNos3</zorder>
   <zorder>pontoEles1</zorder>
   <zorder>pontoEles3</zorder>
   <zorder>pontoEles2</zorder>
   <zorder>cartaJogador1_1</zorder>
   <zorder>cartaJogador1_2</zorder>
   <zorder>cartaJogador1_3</zorder>
   <zorder>cartaJogador2_1</zorder>
   <zorder>cartaJogador2_2</zorder>
   <zorder>cartaJogador2_3</zorder>
   <zorder>cartaJogador3_1</zorder>
   <zorder>cartaJogador3_2</zorder>
   <zorder>cartaJogador3_3</zorder>
   <zorder>sairSala</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
